fe/fe-core/src/main/java/com/starrocks/task/ClearTableTask.java

```diff
@@ -0,0 +1,7 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.task;
+
+public interface ClearTableTask {
```

Comment:
compared to `runnable`, `ClearTableTask` uses `execute()` interface and can return a `boolean` value. However the return value is only checked in UT but not in production code. Can we say that `ClearTableTask` has no vital difference than `runnable`?

Reply:
Yes, you're right.

Reply:
lol.

So the next question is, whether it is necessary to have the `ClearTableTask` refactor? Can both `ClearOlapTableTask` and `ClearLakeTableTask` implements `Runnable` and archive the same goal?

Reply:
Not really necessary now. Actually this patch was extracted from another patch, which already closed now, and the result value did used in that patch and I just keep that change here.